Assets/Scripts/DayCell.cs

```diff
@@ -81,6 +109,61 @@ private bool SetImageActiveDependancy(Image image)
     }
 
 
+    private void TryRemoveCellData()
+    {
+        if (!daycellData.isPoopImageActive && !daycellData.isMedicineImageActive && daycellData.description == "")
```

Comment:
Use string.IsNullOrEmpty(description) rather than comparing to ""

Reply:
Resolved